features/readme.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Accessing WordPress site
+  As a WordPress developer
+  In order to know this Apache is serving static HTML
+  I'd like to check the WordPress readme.html is visible
+
+  @javascript @insulated
```

Comment:
I assume Javascript is to trigger Selenium which we've set up, but what does `insulated` do?